apps/client/src/theme/_viewerDefs.scss

```diff
@@ -26,3 +26,16 @@ $element-border-radius: 8px;
 $timer-color: rgba(white, 80%); // --timer-color-override
 $timer-finished-color: $playback-negative;
 $timer-bold-font-family: "Arial Black", sans-serif; // --card-background-color-override
+$timer-blinking-time: 1.2s;
+
+.blinking {
+  animation: blinking $timer-blinking-time linear infinite;
+}
+
+@keyframes blinking {
+  50% {
+    opacity: 0;
+  }
+}
```

Comment:
not sure the shadowing here is a good idea.
how about the class just be called blink?